### PR TITLE
EFRS-802: Responsive Design: Choice(Edit/Delete/Test) and icon Copy in "Face collection" disappear when reducing screen expansion

### PR DESCRIPTION
**Pull request by Sisha3342**

### CODE REVIEW

**Review by isobolev — COMMENTED** [review]
please add a class to the `apikey` cell, and in styles add media query for 770px with CSS ellipsis styles

File: ui/src/app/features/model-table/model-table.component.html [hunk 1]
```diff
@@ -26,7 +26,13 @@
 
   <ng-container matColumnDef="apiKey">
     <th mat-header-cell *matHeaderCellDef>{{'models.api_key' | translate}}</th>
-    <td mat-cell *matCellDef="let element"> {{element.apiKey}}
+    <td mat-cell *matCellDef="let element">
+      {{windowWidth < 770? (element.apiKey | truncate: 10) : element.apiKey}}</td>
```
Comment (olehkhomyk) [inline ui/src/app/features/model-table/model-table.component.html:30]: Move windowWidth < 770 to method isMobile() and 770 to global const

File: ui/src/app/features/model-table/model-table.component.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@
  * permissions and limitations under the License.
  */
 
-import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import {ChangeDetectionStrategy, Component, EventEmitter, HostListener, Input, Output} from '@angular/core';
```
Comment (isobolev) [inline ui/src/app/features/model-table/model-table.component.ts:17]: unused import

**Review by isobolev — APPROVED** [review]

**Comment by Sisha3342** [comment]
> I have read the CLA Document and I hereby sign the CLA

**Comment by Sisha3342** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Sisha3342** [comment]
recheckcla

**Comment by Sisha3342** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Sisha3342** [comment]
recheckcla

**Pull request merged**